managed/services/agents/mongodb.go

```diff
@@ -38,8 +38,8 @@ type collectorArgs struct {
 var (
 	// New MongoDB Exporter will be released with PMM agent v2.10.0.
 	newMongoExporterPMMVersion = version.MustParse("2.9.99")
-	v2_24_99                   = version.MustParse("2.24.99")
-	v2_25_99                   = version.MustParse("2.25.99")
+	v2_25_0                    = version.MustParse("2.25.0-0")
```

Comment:
I wonder: will this version work with `-0` suffix? Do we have a unit test just in case?

Reply:
Yes, we have a test for it
https://github.com/percona/pmm/blob/main/version/parsed_test.go#L44